packages/ag-charts-community/src/chart/chartProxy.ts

```diff
@@ -216,16 +216,23 @@ export class AgChartInstanceProxy implements AgChartProxy {
     }
 
     private getEnabledOriginators() {
-        const { annotationManager, zoomManager } = this.chart.ctx;
-        const options = this.chart.chartOptions.processedOptions;
+        const {
+            chartOptions: { processedOptions, optionMetadata },
+            ctx: { annotationManager, chartTypeOriginator, zoomManager },
+        } = this.chart;
 
         const originators = [];
 
-        if ('annotations' in options && options.annotations?.enabled) {
+        if ('annotations' in processedOptions && processedOptions.annotations?.enabled) {
             originators.push(annotationManager);
         }
 
-        if (options.navigator?.enabled || options.zoom?.enabled) {
+        const isFinancialChart = optionMetadata.presetType === 'price-volume';
```

Comment:
There may be a better check we can do here?

Reply:
I think this is ok for now.